cmd/autoowners/main.go

```diff
@@ -192,6 +202,8 @@ func getOwnersHTTP(fg FileGetter, orgRepo orgRepo, filenames ownersconfig.Filena
 		}
 		// some OWNERS files (especially upstream ones) can have the '@' symbol in front of usernames, this will not load properly and should be pruned
 		data = []byte(strings.ReplaceAll(string(data), "@", ""))
+		// GitHub allows numeric usernames, but YAML parsers treat unquoted numbers as integers, so we need to quote them
+		data = normalizeNumericUsernames(data)
```

Comment:
This approach is good but I feel an even better solution would be implementing a custom unmarshaler:
```golang
type IntOrString struct {
	data string
}

func (is *IntOrString) UnmarshalJSON(b []byte) error {
	// ...
}

type Owners struct {
	Reviewers []IntOrString `json:"reviewers"`
}
```